instruments/qubitekk/cc1.py

```diff
@@ -120,6 +130,29 @@ def count(self):
     # PROPERTIES #
 
     @property
+    def acknowledge(self):
+        """
+        Gets/sets the acknowledge message state.
```

Comment:
Could you add a little blurb here about what this means for the end user? Something about how setting this to `True` will cause the instrument to echo back all commands sent, `False` will send nothing, and that the default behaviour is false.